packages/zoltan2/src/algorithms/connectivity/Zoltan2_VtxLabel.hpp

```diff
@@ -244,8 +281,8 @@ namespace Kokkos {
 // provided serialization of vtxLabel into char*.
 namespace Teuchos{
 template<typename Ordinal>
-struct SerializationTraits<Ordinal, Zoltan2::IcePropVtxLabel> :
-       public Teuchos::DirectSerializationTraits<Ordinal, Zoltan2::IcePropVtxLabel>
+struct SerializationTraits<Ordinal, Zoltan2::IcePropVtxLabel<Tpetra::Map<>::global_ordinal_type, Tpetra::Map<>::local_ordinal_type> >:
```

Comment:
Everything else in Trilinos has local ordinal first and then global ordinal.  Switching the order for one algorithm is bound to cause trouble someday.

Reply:
I will switch that!